src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const BASE_PRICE = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const amount = days * BASE_PRICE;
+
+  if (days >= LONG_TERM) {
+    return amount - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= SHORT_TERM) {
+    return amount - SHORT_TERM_DISCOUNT;
+  }
```

Comment:
The logic here applies the long-term discount for rentals of 7 or more days, which prevents the short-term discount from being considered. You need to revise the logic to ensure that the correct discount is applied based on the number of days. Consider using an `else if` statement for the short-term discount to ensure that only one discount is applied.